daml/ClaimToken/Service.daml

```diff
@@ -0,0 +1,65 @@
+module ClaimToken.Service where
+
+import ClaimToken.Model
+import DA.Finance.Asset
+
+template Service with
+  operator: Party
+  provider: Party
+  owner: Party
+
+    where
+      signatory operator, provider
+      observer owner
+
+      key (operator, provider, owner) : (Party, Party, Party)
+      maintainer key._1
+    
+      controller owner can 
+        nonconsuming Mint : ContractId AssetDeposit
+          with 
+            assetDepositCid : ContractId AssetDeposit
+            claimTokenR : ClaimTokenR 
+          do
+            originalAssetDeposit <- fetch assetDepositCid
+            assertMsg "provider and owner of provided assetDeposit does not match with the Service Contract used"
+              (originalAssetDeposit.account.owner == owner && originalAssetDeposit.account.provider == provider) -- todo refactor
+            archive assetDepositCid
+            create originalAssetDeposit with asset = (originalAssetDeposit.asset with id = claimTokenR.id)
+
+      controller owner can 
+        nonconsuming OfferTransferRelationship : ContractId TransferRelationshipOffer
+          with 
+            receiver: Party
+          do
+            create TransferRelationshipOffer with sender = owner; ..
+
+      controller owner can
+        nonconsuming Burn : ContractId AssetDeposit
+          with
+            assetDepositCid : ContractId AssetDeposit
+            claimToken : ClaimToken
```

Comment:
Same as above, needs to be `ContractId ClaimToken`

Reply:
@GeorgSchneider same reasoning as https://github.com/digital-asset/da-marketplace/pull/490#discussion_r836173916

Reply:
resolved as https://github.com/digital-asset/da-marketplace/pull/490#discussion_r837065953